CHANGELOG/CHANGELOG-3.0.md

```diff
@@ -0,0 +1,119 @@
+# Release notes for v3.0.0
+
+[Documentation](https://kubernetes-csi.github.io/docs/)
+
+# Changelog since v2.2.0
+
+## Urgent Upgrade Notes 
+
+### (No, really, you MUST read this before you upgrade)
+
+- Update volumeAttachment to v1.
+- RBAC policy was updated to allow the external-attacher to patch VolumeAttachment.Status ([#200](https://github.com/kubernetes-csi/external-attacher/pull/200), [@cwdsuzhou](https://github.com/cwdsuzhou))
+
+## Changes by Kind
+
+### Feature
+
+- Added support for migration of Kubernetes in-tree VMware volumes to CSI. ([#236](https://github.com/kubernetes-csi/external-attacher/pull/236), [@divyenpatel](https://github.com/divyenpatel))
+- Use GA version of CSINode object. The external-attacher now requires Kubernetes 1.17. ([#193](https://github.com/kubernetes-csi/external-attacher/pull/193), [@bertinatto](https://github.com/bertinatto))
+
+### Bug or Regression
+
+- Use dedicated Kubernetes client for leader election that does not get throttled when the external-attacher is under heavy load. ([#242](https://github.com/kubernetes-csi/external-attacher/pull/242), [@jsafrane](https://github.com/jsafrane))
+
+### Uncategorized
```

Comment:
should removal of dep support be here?

Reply:
It will be there once someone regenerates changelog for final 3.0.